test/statement.test.ts

```diff
@@ -78,6 +78,44 @@ CREATE TABLE test_run (counter blurg);
       );
     });
 
+    test("when alter table statement is valid", async function () {
+      {
+        const stmt = db.prepare(`ALTER TABLE ${tableName} ADD COLUMN a int;`);
+        const { results, meta, error } = await stmt.run();
+        deepStrictEqual(results, []);
+        strictEqual(error, undefined);
+        assert(meta.duration != null);
+        assert(meta.txn?.transactionHash != null);
+        strictEqual(meta.txn.name, tableName);
+
+        await meta.txn?.wait();
+      }
```

Comment:
@brunocalza This is kind of a nit, but instead of using scope blocks, we should split these into three different tests.  This takes a little longer to run, but keeps things separated in case there's a problem down the road.
If you're ok with it, since I have a working local setup, I can make that change and potentially add a test the ensures only the table owner can alter a table?